Formula/g/goimports.rb

```diff
@@ -0,0 +1,27 @@
+class Goimports < Formula
+  desc "Go formatter that additionally inserts import statements"
+  homepage "https://pkg.go.dev/golang.org/x/tools/cmd/goimports"
+  url "https://github.com/golang/tools/archive/refs/tags/v0.31.0.tar.gz"
+  sha256 "e5d74f1e63a1ee669e75e76668cea1b110e2b9d19c67710f60939ee38070a5a7"
+  license "BSD-3-Clause"
+
```

Comment:
based on https://github.com/golang/tools/tags it seems it needs a livecheck block to not use `gopls/*` tags:

```suggestion

  livecheck do
    url :stable
    regex(/^v?(\d+(?:\.\d+)+)$/i)
  end

```

Reply:
Good catch, thanks. It was working without it on my personal tap because the `vX.Y.Z` tag somehow end up being the latest 😅